modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -209,6 +209,11 @@ Events:     <11>
 <10> The pods on the node.
 <11> The events reported by the node.
 
+[NOTE]
+====
+The control plane label is not automatically added to newly created or updated master nodes. If you want to use the control plane label for your nodes, you can manually configure the label. See xref:../../nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes] for more information about manually configuring labels.
```

Comment:
Inclusive language suggest that we use "main" or "control plane node" unless you've been given permission to use "master." https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language. Also, swap .html with .adoc. 

```suggestion
The control plane label is not automatically added to newly created or updated control plane nodes. If you want to use the control plane label for your nodes, you can manually configure the label. See xref:../../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes] for more information about manually configuring labels.
```

Reply:
@stevsmit Thanks! I've updated the "html" to "adoc". However, this addition needs to use the "master" language because it references nodes that were previously created with the master label. Both master and control plane labels are still functional. It is up to the user to manually make the change if they want to do so.